### PR TITLE
Plots for fractions of fake added, using MC info

### PR DESCRIPTION
**Pull request by chiarazampolli**
@njacazio , @noferini , FYI.

### CODE REVIEW

File: Modules/TOF/src/TOFMatchedTracks.cxx [hunk 1]
```diff
@@ -122,9 +156,17 @@ void TOFMatchedTracks::monitorData(o2::framework::ProcessingContext& ctx)
     }
     // loop over TOF MatchInfo
     for (const auto& matchTOF : mTPCTOFMatches) {
-      const auto& trk = mTPCTracks[matchTOF.getTrackRef().getIndex()];
+      GTrackID gTrackId = matchTOF.getTrackRef();
+      const auto& trk = mTPCTracks[gTrackId.getIndex()];
       mMatchedTracksPt[trkType::UNCONS]->Fill(trk.getPt());
       mMatchedTracksEta[trkType::UNCONS]->Fill(trk.getEta());
+      if (mUseMC) {
+	auto lbl = mRecoCont.getTrackMCLabel(gTrackId);
```
Comment (njacazio) [inline Modules/TOF/src/TOFMatchedTracks.cxx:164]: Hi @chiarazampolli indeed here probably the formatting CI  fails

**Review by njacazio — APPROVED** [review]
Hi @chiarazampolli I think the code is good to go! Just the CI fixes are needed

File: Modules/TOF/src/TOFMatchedTracks.cxx (hunk 1 above)
Comment (chiarazampolli) [reply Modules/TOF/src/TOFMatchedTracks.cxx]: Yes, I also noticed, sorry, I pushed now the fix.

**Comment by chiarazampolli** [comment]
Now clang-format complains about different code... `Modules/MFT/include/MFT/QcMFTClusterTask.h`

**Comment by chiarazampolli** [comment]
Unless I overlooked, the errors are not related to this PR.

**Comment by Barthelemy** [comment]
I do not understand why clang-format complains
merging

**Pull request merged**